### PR TITLE
feat(adr): use github as main hub for the Opinionated Digital Center

### PR DESCRIPTION
**Pull request by esciara**

### CODE REVIEW

**Comment by slequeux** [comment]
To discuss in a next meeting I think :)

**Comment by esciara** [comment]
yes... I wanted to share the information I put in the ADR. Forgot to mention it :)

File: docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+## Context and Problem Statement
+
+We want to share the Opinionated Digital Center's (ODC) opinions, code, templates,
+pipelines and tools with the community. We want to make it as easy as possible to access
+and to contribute to, either by exchanging ideas, submitting issues and Pull/Merge
+Requests. Hence, we need to choose a hub.
+
+## Decision Drivers
+
+* Our main audience target are environments and companies using self-hosted or
```
Comment (slequeux) [inline docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md]: `environments` ?

File: docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  cloud versions of GitLab or GitHub for hosting the code, and GitLab CI for pipelines.
```
Comment (slequeux) [inline docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md]: Also Jenkins for pipelines. Even if we explicitely tell why we will not do anything special for Jenkins

File: docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+## Pros and Cons of the Options
+
+### Main hub: GitLab; pipeline: GitLab CI; mirroring: GitHub, pull only
+
+* Good, because:
+  * Everything is in one place and fully integrated.
+* Bad, because:
+  * GitLab's popularity amongst the community is not as high as GitHub's.
+    Even our contributors, who are using GitLab daily (and liking it) at their workplace
+    turn first to GitHub for finding code and, most importantly, interacting around it.
```
Comment (slequeux) [inline docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md]: I would add that this opinion is mostly amongst open source community.
The first name that comes to mind while looking for OS projects is GitHub

File: docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md (hunk 1 above)
Comment (esciara) [reply docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md]: organisations.

File: docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md (hunk 2 above)
Comment (esciara) [reply docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md]: I added stuff, let me know what you think about it.

File: docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md (hunk 3 above)
Comment (esciara) [reply docs/adr/XXXX-use-github-as-main-hub-for-the-opinionated-digital-center.md]: done

**Pull request merged**